docs/en/dora/Metadata.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: global
+title: Metadata Cache and Invalidation in Dora
+group: Dora
+priority: 3
+---
+
+* Table of Contents
+  {:toc}
+
+In Dora, Alluxio caches metadata and data on workers. This document explains how the metadata is cached and how the metadata is invalidated and refreshed. Alluxio uses Paged data cached. Along with the metadata invalidation, the data is also invalidated.
+
+## Two Different Levels of Metadata Cache
+
+In Dora, metadata is cached on workers. There are two levels of metadata cache: in-memory cache and in-RocksDB cache.
+Metadata in RocksDB can be considered as off-heap and persistent cache. It is stored in persistent storage, e.g. local file system.
+The metadata cached in memory will be lost if worker restarts. On contrary, the metadata cached in RocksDB is persistent, and can be loaded
+back when worker restarts. Alluxio uses config keys to control the capacity of the cache and the life of the cache. These config values should
+be tuned according to available memory resources.
+
+### Property Keys for In-memory Cache
+The following keys are used to control the size of in-memory metadata cache from getStatus() result, and the life (Time To Live, TTL) of such metadata.
+- `DORA_UFS_FILE_STATUS_CACHE_SIZE (alluxio.dora.ufs.file.status.cache.size)`: the maximum number of metadata items cached in memory.
```

Comment:
Can we give an estimation of how much memory each cached item takes up? so that users can estimate the amount of memory needed, given the number of files of their workload.

Reply:
Good suggestion.
Let me add some estimation here.